ext/dramsys/README

```diff
@@ -1,11 +1,13 @@
-Follow these steps to get DRAMSys as part of gem5
+Follow these steps to build DRAMSys as part of gem5
 
 1. Go to ext/dramsys (this directory)
 2. Clone DRAMSys: 'git clone https://github.com/tukl-msd/DRAMSys DRAMSys'
-3. Change directory to DRAMSys: 'cd DRAMSys'
-4. Checkout the correct commit: 'git checkout -b gem5 09f6dcbb91351e6ee7cadfc7bc8b29d97625db8f'
-5. Recursively obtain the submodules: 'git submodule update --init --recursive'
+
+The latest verified working version is 8e021eada2cb937d5be24dbd02f3980e2193d6ec.
```

Comment:
Any chance you can point to a tag or stable release?

Reply:
Hi Jason, thanks for the quick review.
We will add a tag to make things easier for users.

Have you any ideas about the build type setting in scons?

Reply:
No, I have no idea how to get build information. gem5's scons build is crazy magic to me.

I guess what you're kinda going for would be "gem5.fast" would be like release and "gem5.opt" or "gem5.debug" would be "RelWithDebInfo" or "Debug", maybe? gem5 doesn't really have a "release" vs "debug" build. We just have a build with/without optimizations and with/without debug symbols.

Reply:
Updated the readme to point to a tag.

Sadly, setting the correct CMake build type seems impossible as the external modules are included before the gem5 debug/opt/fast logic is run.

Reply:
No worries. I don't think it's terribly important to get it perfect.